src/ui/controls/edit.cpp

```diff
@@ -17,32 +17,23 @@
  * along with this program. If not, see http://gnu.org/licenses
  */
 
-
 #include "ui/controls/edit.h"
 
-#include "common/config.h"
-
 #include "app/app.h"
```

Comment:
`config.h` needs to be placed before `app.h`. Also remember to separate them with empty line.